javascript-timers/interval.js

```diff
@@ -0,0 +1,11 @@
+setInterval(countdown, 1000);
+var count = 4;
+clearInterval(countdown);
```

Comment:
Did this work? I would think that as a result, the code would never run. You should be clearing the interval inside the `countdown` function after you show the final text

Reply:
Yeah it still worked, But I will change it right now